apps/next/util/ssr/onLoad.ts

```diff
@@ -0,0 +1,52 @@
+import { BannerRefreshService } from '@app/services/BannerRefreshService';
+import { ToadScheduler, SimpleIntervalJob, AsyncTask } from 'toad-scheduler';
+import { logger } from '../logger';
+import prisma from './prisma';
+
+let hasLoaded = false;
+
+// Function is called when prisma is loaded, which hopefully will be almost right at startup
+export function onLoad(): void {
+    if (hasLoaded === true) {
+        return;
+    }
+
+    try {
+        hasLoaded = true;
+        const scheduler = new ToadScheduler();
+
+        const refreshProfessional = new AsyncTask(
+            'refresh professional',
+            async () => {
+                const refreshService = new BannerRefreshService(prisma, logger);
+                await refreshService.refreshBanners('Professional');
+            },
+            (err: Error) => {
+                /* handle error here */
+                logger.error('Error occured executing scheduled job.', { ...err });
+            }
+        );
+
+        // const refreshPersonal = new AsyncTask('refresh personal',
```

Comment:
Assume we don't need this?

Reply:
Well I was going to uncomment it when after testing it, but maybe we should uncomment it now.